commands/common/cmd_commons_test.go

```diff
@@ -1,27 +1,20 @@
-package commands
+//go:build test
+// +build test
```

Comment:
I guess we don't need this way of setting build tags anymore
```suggestion
```

Reply:
BTW, why do we need build tag for tests?

Reply:
Go is already supposed to exclude unused code from the binary, no need for a build tag.